addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5022,7 +5022,17 @@ msgctxt "#10057"
 msgid "Edit file source"
 msgstr ""
 
-#empty strings from id 10058 to 10098
+#: xbmc/video/windows/CGUIWindowVideoNav.cpp
+#: xbmc/video/dialogs/CGUIDialogVideoInfo.cpp
+msgctxt "#10058"
+msgid "Remove tag from library"
+msgstr ""
+
+#. xbmc/video/windows/CGUIWindowVideoNav.cpp
+msgctxt "#10059"
+msgid "Would you really like to remove tag '{0:s}' from the library ?"
+
+#empty strings from id 10060 to 10098
```

Comment:
With suggested changes to `CGUIWindowVideoNav::OnDeleteItem` these become unnecessary

Reply:
Done.